api/base/urls.py

```diff
@@ -22,7 +22,8 @@
                 url(r'^files/', include('api.files.urls', namespace='files')),
                 url(r'^docs/', include('rest_framework_swagger.urls')),
                 url(r'^institutions/', include('api.institutions.urls', namespace='institutions')),
-                url(r'^collections/', include('api.collections.urls', namespace='collections'))
+                url(r'^collections/', include('api.collections.urls', namespace='collections')),
+                url(r'^guid/', include('api.guid.urls', namespace='guid'))
```

Comment:
Despite what I said on Jira. That guy clearly had no idea what he was talking about.